frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-single-namespace.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { checkErrors, testName } from '../../../integration-tests-cypress/support';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+const operatorName = '3scale Operator';
+
+describe(`Interacting with a single namespace install mode Operator (${operatorName})`, () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const operatorPkgName = '3scale-community-operator';
```

Comment:
We might consider installing a custom catalog source like we were before to lock to a specific operator version, but I think this is fine for now. We'll see if it becomes an issue.